doc/release/1.14.0-notes.rst

```diff
@@ -92,6 +92,9 @@ Future Changes
   This translation now gives a warning that explains what translation is
   occurring.  In the future, the translation will be disabled, and the first
   example will be made equivalent to the second.
+This translation now gives a warning explaining what translation is occurring.
```

Comment:
Wrong indentation.

Reply:
This entire block duplicates the lines above it...

Reply:
> This entire block duplicates the lines above it...

Not sure how this happened. Will remove this from the PR.